packages/core/strapi/tests/api/relations.test.api.js

```diff
@@ -132,6 +133,86 @@ const updateEntry = async (pluralName, id, data, populate) => {
   return body.data;
 };
 
+const createShop = async ({
```

Comment:
Imo these helpers make the tests more readable and short. 
If we like this structure, I would like to change tests from this file to use it.